Std/Data/Bool.lean

```diff
@@ -38,6 +38,50 @@ instance (x y : Bool) : Decidable (x < y) := inferInstanceAs (Decidable (!x && y
 instance : Max Bool := ⟨or⟩
 instance : Min Bool := ⟨and⟩
 
+theorem false_ne_true : false ≠ true := Bool.noConfusion
+
+theorem eq_false_or_eq_true : (b : Bool) → b = true ∨ b = false := by decide
+
+theorem eq_false_iff : {b : Bool} → b = false ↔ b ≠ true := by decide
+
+theorem eq_true_iff : {b : Bool} → b = true ↔ b ≠ false := by decide
```

Comment:
Sorry for the delay, what's the motivation for these four lemmas?  It seems like they all reduce or simp or otherwise handled through tactics.

Reply:
I think some of these are moved from `Std.Logic` but `eq_true_iff` is not as useful as `ne_false_iff`.